packages/ui-react/src/containers/AccountModal/forms/PersonalDetails.tsx

```diff
@@ -40,10 +40,12 @@ const PersonalDetails = () => {
   );
 
   const nextStep = useCallback(() => {
-    const hasOffers = accessModel === ACCESS_MODEL.SVOD || (accessModel === ACCESS_MODEL.AUTHVOD && hasMediaOffers);
+    const hasSubscriptionPlansOrOffers = !!subscriptionOffers.length;
+
+    const hasOffers = accessModel === ACCESS_MODEL.SVOD || (accessModel === ACCESS_MODEL.AUTHVOD && hasMediaOffers) || hasSubscriptionPlansOrOffers;
```

Comment:
Are you sure this change is needed? When the access model is SVOD, we can already assume that there are subscription offers that will be fetched in the next step. Or are we not configuring global SVOD plans in the new setup?

Reply:
There might be no subscription offers yet, in which case it makes sense to skip the `choose-offer` modal and go straight to `welcome`.